components/aws/sagemaker/tests/unit_tests/tests/commonv2/test_spec_input_parsers_v2.py

```diff
@@ -0,0 +1,50 @@
+import argparse
+import unittest
+
+from commonv2.spec_input_parsers import SpecInputParsers as par
+
+
+class SpecInputParsersTestCase(unittest.TestCase):
+    def test_yaml_or_json_list(self):
+        self.assertIsNone(par.yaml_or_json_list(""))
+        self.assertEqual([], par.yaml_or_json_list("[ ]"))
+        self.assertEqual(
+            [{"a": 1}, {"b": 2}], par.yaml_or_json_list('[ {"a": 1}, {"b": 2} ]')
+        )
+
+        with self.assertRaises(argparse.ArgumentTypeError):
+            par.yaml_or_json_list("{a: 1}")
+
+        self.assertEqual(
+            [{"a": 1}, {"b": 2}],
+            par.yaml_or_json_list(
+                """
```

Comment:
formatting here looks a little strange is this what we see after running black?

Reply:
Yea its the same